webpack.config.js

```diff
@@ -55,6 +55,17 @@ module.exports = env => ({
             }
           }
         ]
+      },
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          // Creates `style` nodes from JS strings
+          "style-loader",
+          // Translates CSS into CommonJS
```

Comment:
Also, bless you for adding these comments, I can never for the life of me remember what these loaders do